test/plugins/test_titlecase.py

```diff
@@ -129,8 +129,34 @@ def test_seperators(self):
             ),
         ]
         for testcase in testcases:
-            seperators, given, expected = testcase
-            with self.configure_plugin({"seperators": seperators}):
+            separators, given, expected = testcase
+            with self.configure_plugin({"separators": separators}):
+                assert TitlecasePlugin().titlecase(given) == expected
+
+    def test_all_caps(self):
```

Comment:
This seems like a prime example of where a `pytest.parametrize` would be be my preferred syntax. We should spent some time in the future to fully migrate our test setup to pytest :/